shared/remediations/bash/templates/remediation_functions

```diff
@@ -778,7 +778,7 @@ function replace_or_append {
 
   # If there is no print format specified in the last arg, use the default format.
   if ! [ "x$format" = x ] ; then
-    printf -v formatted_output $format $stripped_key $value
+    printf -v formatted_output "$format" $stripped_key $value
```

Comment:
Makes sense to me. ACK.

Will test the proper work of the changed remediations yet.